core/templates/dev/head/pages/footer.html

```diff
@@ -16,7 +16,6 @@ <h4 translate="I18N_FOOTER_TEACH_LEARN_ALL_CAPS"></h4>
           <li><a href="/get_started" translate="I18N_FOOTER_GET_STARTED"></a></li>
           <li><a href="/teach#playbook" translate="I18N_FOOTER_PARTICIPATION_PLAYBOOK"></a></li>
           <li><a href="/library" translate="I18N_FOOTER_BROWSE_LIBRARY"></a></li>
-          <li><a href="/forum" translate="I18N_FOOTER_FORUM"></a></li>
```

Comment:
This can be left as is, actually; just change the link. We're just getting rid of /forum, not the forum itself.

Reply:
Added "https://...." link. (Done!)